src/tokens-studio/spectrum2-colors/spectrum2/component/dark.json

```diff
@@ -269,7 +269,7 @@
           }
         }
       },
-      "gripper-drag": {
+      "gripper-color-drag": {
```

Comment:
@larz0 This would be the first color token with the word 'color' in it. Notice: This is not the spectrum-tokens name, but the name for the token in figma and the variables. Please put the spectrum-tokens name into the spectrum-tokens object, e.g. here: https://github.com/adobe/spectrum-tokens-studio-data/pull/168/files#diff-1b6a253c366c334e1436d57bc36dc0e6083a2300ce2c823b8a68c1cca5ef4017L278

Reply:
In Spectrum Tokens we would want the word `color` to be in there because it's the property it's changing. I guess we could use `fill-color` or `background-color` but `color` alone is fine. As far as what the name should be in Tokens Studio data, that's up to @nabuhasan and @lynnhao to approve or ask to change.